docs/source/index.rst

```diff
@@ -37,17 +37,22 @@ set ``use_pandas=True`` in functions that accept that parameter.  To install
    pip install pandas
 
 Machine learning features in the ``ml`` namespace have a soft dependency on
-``scikit-learn`` and ``pandas``. Install ``scikit-learn`` to
+``scikit-learn``, ``pandas``, ``civisml-extensions``, and
+``glmnet``. Install ``scikit-learn`` to
```

Comment:
I think it's not quite right to call our relationship with `civisml-extensions` and `glmnet` a soft dependency. Installing those won't really change what the API client code will do for you, while installing, say, scikit-learn does add new capabilities.

I think it's better to have a separate paragraph noting that various CivisML models have open-source dependencies beyond scikit-learn, and that if users want to download their model, they would need to install those dependencies. The full description is probably more appropriate for `ml.rst`, but a mention here isn't a bad idea. 

If you're documenting these libraries, you should also include `muffnn`. The MLP models use that library.